examples/informers/ServiceCollectionKubernetesExtensions.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Linq;
+using System.Threading;
+using k8s;
+using k8s.Informers;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Rest;
+using Microsoft.Rest.TransientFaultHandling;
+
+namespace informers
+{
+    public static class Extensions
+    {
+        public static IServiceCollection AddKubernetes(this IServiceCollection services)
+        {
+            services.AddHostedService<ControllerService>();
+            var controllers = AppDomain.CurrentDomain.GetAssemblies()
+                .SelectMany(x => x.DefinedTypes)
+                .Where(x => x.IsClass && !x.IsAbstract && typeof(IController).IsAssignableFrom(x))
+                .ToList();
+            foreach (var controller in controllers) services.AddSingleton(typeof(IController), controller);
+            var config = KubernetesClientConfiguration.BuildDefaultConfig();
+
+            services.AddSingleton(config);
+            services.AddSingleton<RetryDelegatingHandler>();
```

Comment:
Where is this used if it's being created further down manually?

Reply:
Left over from later refactoring. Removed now